### PR TITLE
Translation to Portuguese

### PR DESCRIPTION
**Pull request by bigbruno**
Hope it's useful :)

### CODE REVIEW

**Comment by austin987** [comment]
Thanks, but it's failing the unit tests (./tests/-shell-checks). Could you please take a look at that?

**Comment by bigbruno** [comment]
I think this is correct now.

File: src/winetricks [hunk 1]
```diff
@@ -11766,6 +11880,7 @@ load_quicktime76()
 
         case ${LANG} in
             ru*) w_warn "В настройках Quicktime включите Дополнительно / Безопасный режим (только gdi), иначе видеофайлы не будут воспроизводиться." ;;
+            *) w_warn "Nas preferências do Quicktime, marque Advanced / Safe Mode (gdi), ou os vídeos não irão reproduzir." ;;
```
Comment (austin987) [inline src/winetricks:11883]: This should be `pt*`

**Review by austin987 — CHANGES_REQUESTED** [review]

File: src/winetricks (hunk 1 above)
Comment (bigbruno) [reply src/winetricks]: Thanks, corrected

File: src/winetricks [hunk 2]
```diff
@@ -867,6 +897,9 @@ w_read_key()
             pl*) _W_keymsg="Proszę podać klucz dla programu '${W_PACKAGE}'"
                 _W_nokeymsg="Nie podano klucza"
                 ;;
+            pt*)  _W_keymsg="Por favor, insira a chave do aplicativo '${W_PACKAGE}'"
```
Comment (austin987) [inline src/winetricks:900]: Please follow the existing whitespace convention (one space after the paren, not two).

(Similar issues below).

File: src/winetricks [hunk 3]
```diff
@@ -2887,6 +2923,7 @@ w_workaround_wine_bug()
         da*) w_warn "Arbejder uden om wine-fejl ${1} ${_W_msg}" ;;
         de*) w_warn "Wine-Fehler ${1} wird umgegangen ${_W_msg}" ;;
         pl*) w_warn "Obchodzenie błędu w wine ${1} ${_W_msg}" ;;
+        pt*)   w_warn "Trabalhando em torno do bug do wine ${1} ${_W_msg}" ;;
```
Comment (austin987) [inline src/winetricks:2926]: Please remove the extra spaces before `w_warn`.

File: src/winetricks [hunk 4]
```diff
@@ -3634,6 +3676,17 @@ winetricks_prefixmenu()
              _W_msg_unattended1="Włącz cichą instalację"
              _W_msg_help="Wyświetl pomoc"
              ;;
+        pt*)   _W_msg_title="Winetricks - Escolha um wineprefix"
```
Comment (austin987) [inline src/winetricks:3679]: Here.

File: src/winetricks [hunk 5]
```diff
@@ -3740,6 +3794,10 @@ winetricks_mkprefixmenu()
              _W_msg_name="Name"
              _W_msg_arch="Architektur"
              ;;
+        pt*)   _W_msg_title="Winetricks - criar novo wineprefix"
```
Comment (austin987) [inline src/winetricks:3797]: Here.

File: src/winetricks [hunk 6]
```diff
@@ -3811,6 +3869,20 @@ winetricks_mainmenu()
              _W_msg_folder='Przeglądać pliki'
              _W_msg_annihilate="Usuńąć WSZYSTKIE DANE I APLIKACJE WEWNĄTRZ TEGO PREFIKSU WINE"
              ;;
+        pt*)   _W_msg_title="Winetricks - o prefixo atual é \"${WINEPREFIX}\""
```
Comment (austin987) [inline src/winetricks:3872]: Here.

File: src/winetricks [hunk 7]
```diff
@@ -3951,6 +4023,9 @@ winetricks_settings_menu()
         pl*) _W_msg_title="Winetricks - obecny prefiks to \"${WINEPREFIX}\""
              _W_msg_body='Jakie ustawienia chcesz zmienić?'
              ;;
+        pt*)   _W_msg_title="Winetricks - o prefixo atual é \"${WINEPREFIX}\""
```
Comment (austin987) [inline src/winetricks:4026]: Here.

File: src/winetricks [hunk 8]
```diff
@@ -4125,6 +4212,10 @@ winetricks_showmenu()
              _W_msg_body='Które paczki chesz zainstalować?'
              _W_cached="zarchiwizowane"
              ;;
+        pt*)   _W_msg_title="Winetricks - o prefixo atual é \"${WINEPREFIX}\""
```
Comment (austin987) [inline src/winetricks:4215]: Here.

File: src/winetricks [hunk 9]
```diff
@@ -4498,6 +4605,7 @@ winetricks_list_all()
         da*) _W_cached="cached"   ; _W_download="kan hentes"    ;;
         de*) _W_cached="gecached" ; _W_download="herunterladbar";;
         pl*) _W_cached="zarchiwizowane"   ; _W_download="do pobrania"  ;;
+        pt*)   _W_cached="em cache"   ; _W_download="para download"  ;;
```
Comment (austin987) [inline src/winetricks:4608]: Here.

File: src/winetricks [hunk 10]
```diff
@@ -5728,6 +5840,7 @@ winetricks_install_app()
         da*) fail_msg="Installationen af pakken $1 fejlede" ;;
         de*) fail_msg="Installieren von Paket $1 gescheitert" ;;
         pl*) fail_msg="Niepowodzenie przy instalacji paczki $1" ;;
+        pt*)   fail_msg="Falha ao instalar o pacote $1" ;;
```
Comment (austin987) [inline src/winetricks:5843]: Here.

**Review by austin987 — COMMENTED** [review]
Thanks, now that CI passes, I took a closer look.

Looks mostly good, just some whitespace issues that I've marked. Once that's fixed, we can squash and merge :).

File: src/winetricks (hunk 3 above)
Comment (bigbruno) [reply src/winetricks]: Ok

**Review by austin987 — APPROVED** [review]

**Pull request merged**